include/bout/mesh.hxx

```diff
@@ -486,10 +491,12 @@ public:
   virtual void addBoundary(BoundaryRegion* UNUSED(bndry)) {}
 
   /// Get all the parallel (Y) boundaries on this processor
```

Comment:
Please could you expand this docstring to mention the effect of `type` -- the message on your first commit is a good example

Reply:
I replaced the doc string